frontend/src/pages/Login.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useContext, useCallback } from "react";
+import { useHistory, useLocation } from "react-router-dom";
+import { AxiosInstance } from '../utils/Helpers'
+import AuthContext from "../context/AuthContext";
+import Button from '../components/elements/Button';
+import Input from '../components/elements/Input';
+import styled from "styled-components";
+
+
+const Login = ({setError, setErrorMessage, setSuccess, setSuccessMessage}) => {
+  const [credentials, setCredentials] = useState({});
+  const { setLoggedIn } = useContext(AuthContext)
+  let history = useHistory();
+  let location = useLocation();
```

Comment:
Why `let`?

Reply:
you are amazing, thank you for all of this, I don't know how you make the time <3

(because that's what the example in the documentation used and I didn't think it through, thanks for pointing it out!)